administrator/language/en-GB/lib_joomla.ini

```
@@ -454,6 +454,7 @@ JLIB_HTML_NO_RECORDS_FOUND="No records found."
 JLIB_HTML_PAGE_CURRENT="Page %s"
 JLIB_HTML_PAGE_CURRENT_OF_TOTAL="Page %s of %s"
 JLIB_HTML_PAGINATION="Pagination"
+JLIB_HTML_PAGINATION_NUMBERS="%d - %d / %d items"
```

```suggestion
JLIB_HTML_PAGINATION_NUMBERS="%1$d- %2$d/ %3$d items"
```

Without this translators are unable to change the order of the variables which might be required in their language

---

I fixed the typo here too and merged it thanks